pkg/arvo/neo/cod/std/src/fil/eyre.md

```diff
@@ -0,0 +1,77 @@
+# Eyre
+
+Eyre is Urbit's HTTP server vane. Shrubbery can use Eyre to serve UIs to your device and serve resources of all kinds to the clearweb. 
+
+## Types
+
+[looks like it stores state in two diff formats and has a map of bound URLs]
+
+The Eyre shane stores three things in its state.
+- `by-id`: ???
```

Comment:
`by-id` map from request ID to the path of the shrub handling said request, and `by-pith` is just `by-id` backward